partiql-lang/src/test/kotlin/org/partiql/lang/syntax/PartiQLParserExecTest.kt

```diff
@@ -0,0 +1,78 @@
+package org.partiql.lang.syntax
+
+import com.amazon.ion.Decimal
+import com.amazon.ionelement.api.ionDecimal
+import com.amazon.ionelement.api.ionInt
+import com.amazon.ionelement.api.ionString
+import org.junit.Test
+import org.partiql.lang.domains.id
+
+/**
+ * Tests parsing of EXEC using just the PIG parser.
+ */
+class PartiQLParserExecTest : PartiQLParserTestBase() {
+    override val targets: Array<ParserTarget> = arrayOf(ParserTarget.DEFAULT)
+
+    // ****************************************
+    // EXEC clause parsing
+    // ****************************************
+    @Test
```

Comment:
(self-review) `EXEC` parsing tests were copied directly from `PartiQLParserTest` and runs on just the PIG parser.